docs/topics/api/accounts.rst

```diff
@@ -72,7 +72,65 @@ This endpoint is a shortcut to your own account. It returns an :ref:`account obj
 
 .. http:get:: /api/v3/accounts/profile/
 
-    .. _self-account-object:
+
+----------------
+Collections List
+----------------
+
+.. _collection-list:
+
+.. note:: This API requires :doc:`authentication <auth>`.
+
+This endpoint allows you to list all collections authored by this user.
+Access is limited to your own collections, or users with `Users:Edit` permission.
```

Comment:
reworded in https://github.com/mozilla/addons-server/pull/5358/commits/1aa9eaaf0c0fc9554bdd4f2083e5ef0adae5d3b6 but I stuck with "can only list" rather than "can only access".